docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
+CodeChecker checkers --profile sensitive --details
+```
+commands.
+
+The `./reports` directory is the "database" of CodeChecker that allows to
+manage further working steps.
+
+### View the analysis results in the command line<a name="view-analysis-results"></a>
 ```sh
 CodeChecker parse ./reports
 ```
 
 Hint:
- You can do the 1st and the 2nd step in one round by executing `check`
+You can do the 1st and the 2nd step in one round by executing `check`
 ```sh
-cd tmux
+cd <repo root dir>/docs/examples
 make clean
-CodeChecker check -b "make" -o ./reports
+rm -rf ./reports
+CodeChecker check --build "make" --output ./reports --enable-all
 ```
-or to run on 22 threads
+or to run on 22 threads both the compilation and the analysis:
+
 ```sh
-CodeChecker check -j22 -b "make clean;make -j22" -o ./reports
+CodeChecker check --jobs 22 --build "make clean ; make --jobs 22" \
+    --output ./reports --enable-all
 ```
 
-### Cross-Compilation <a name="cross-compilation"></a>
+### Cross-Compilation<a name="cross-compilation"></a>
 Cross-compilers are auto-detected by CodeChecker, so the `--target` and the
-compiler pre-configured
-include paths of `gcc/g++` are automatically passed to `clang` when analyzing.
+compiler pre-configured include paths of `gcc/g++` are automatically passed to
+`clang` when analyzing.
 
 **Make sure that the compilers used for building the project (e.g.
 `/usr/bin/gcc`) are accessible when `CodeChecker analyze` or `check` is
 invoked.**
 
-### Incremental Analysis <a name="incremental-analysis"></a>
+## Step 3: View analysis results in command line or generate static HTML files<a name="step-3"></a>
+You can print detailed results (including the control flow) in command line by
+running:
+
+```sh
+CodeChecker parse --print-steps ./reports
+...
+Found no defects in divide.c
+[STYLE] .../docs/examples/src/main.c:15:7: do not use 'else' after 'return' [readability-else-after-return]
+    } else {
+      ^
+  Report hash: c120e732461d8e478c35d1940ef53488
+  Steps:
+    1, main.c:15:7: do not use 'else' after 'return'
+
+[STYLE] .../docs/examples/src/main.c:16:43: 10 is a magic number; consider replacing it with a named constant [readability-magic-numbers]
+        test_case = strtol(argv[1], NULL, 10);
+                                          ^
+  Report hash: 17e5ce988ab2cfd6284d3ec7c66904a0
+  Steps:
+    1, main.c:16:43: 10 is a magic number; consider replacing it with a named constant
+
+Found 2 defect(s) in main.c
+...
+```
+
+It is possible to generate reports as plain `HTML` files using the
+`CodeChecker parse` command. (For other output formats please consult with help
+of `parse` subcommand.)
+
+```sh
+CodeChecker parse --export html --output ./reports_html ./reports
+...
+To view the results in a browser run:
+> firefox ./reports_html/index.html &
+```
+
+`./reports_html` directory will contain an `index.html` with a link to all
+findings that are stored in separate `HTML` files (one per analyzed build
+action).
```

Comment:
It will contain 1 html file for each plist file and not for each build action.

Reply:
Done.